src/teleop/gui/src/components/Carousel.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+<div>
+    <h3>Carousel Data</h3>
+    <div class="box1">
+        <Checkbox ref="open-loop" :name="'Open Loop'" @toggle="openLoop = !openLoop" />
+        <div class="controls">
+            <div v-if="!openLoop">
+                <label for="position">Rotate carousel to position: </label>
+                <select v-model="site">
```

Comment:
Same thing here as with the general site selection component, Radio buttons would be nice.